src/rendering/renderable.cpp

```diff
@@ -75,15 +75,6 @@ documentation::Documentation Renderable::Documentation() {
         "Renderable",
```

Comment:
Missing codegen for this documentation? Or is there an issue with the base class that causes problems with using codegen here?

Reply:
Nope, no real reason.    We only ran the codegen on the `module` folder before, now it is the `module` and the `src` file.  It takes about 600ms in Debug longer to compile, but I think that's acceptable for now